C/plugins/utils/get_plugin_info.cpp

```diff
@@ -41,22 +41,23 @@ int main(int argc, char *argv[])
     exit(1);
   }
 
+  openlog("Fledge PluginInfo", LOG_PID|LOG_CONS, LOG_USER);
+  setlogmask(LOG_UPTO(LOG_WARNING));
+
   if (access(argv[1], F_OK|R_OK) != 0)
   {
-    fprintf(stderr, "Unable to access library file '%s', exiting...\n", argv[1]);
+    syslog(LOG_ERR, "Unable to access library file '%s', exiting...\n", argv[1]);
```

Comment:
May be retain fprintf to stderr also (in addition to syslog) so that errors can be seen on console also.